test/transaction.promise.test.js

```diff
@@ -176,7 +176,8 @@ describe('transactions with promise', function() {
     it('should rollback the transaction if timeout', function(done) {
       Post.find({where: {title: 't3'}}, {transaction: currentTx},
         function(err, posts) {
-          if (err) return done(err);
+          should.exist(err);
```

Comment:
This change is suspicious. We expect to rollback an active transaction upon timeout.

Reply:
@raymondfeng yeah I know:
My local tests with mysql 5.6.17 passes this test, **only CI fails**, that's why I make this change.

And according to [this line](https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/dao.js#L153) and [the ensureActive method](https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/transaction.js#L201), an error should be thrown here. 

Any suggestion?

Reply:
never mind I think it's related to the `--exit` flag :p